dev

```diff
@@ -3,14 +3,28 @@
 set -e
 
 DIR=$(pwd)
-MESG_LOG_LEVEL=${MESG_LOG_LEVEL:-debug}
+
+# mesg config variables
+MESG_NAME=${MESG_NAME:-engine}
+MESG_PATH=${MESG_PATH:-$HOME/.mesg}
+MESG_SERVER_ADDRESS=${MESG_SERVER_ADDRESS:-:50052}
+MESG_LOG_FORMAT=${MESG_LOG_FORMAT:-text}
 MESG_LOG_FORCECOLORS=${MESG_LOG_FORCECOLORS:-false}
+MESG_LOG_LEVEL=${MESG_LOG_LEVEL:-debug}
+
 VERSION=local
 LDFLAGS="-X 'github.com/mesg-foundation/core/version.Version=$VERSION'"
 LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceEndpoint=https://ropsten.infura.io/v3/6115ae2531f64c04a9a392cf500e5fbe'"
 LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceAddress=0xeCC1A867F871323350A1A89FcAf69629a2d5085e'"
 LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceToken=0x5861b3dc52339d4f976b7fa5d80db6cd6f477f1b'"
+
+# if set to true the docker with system services will be started.
+MESG_SERVICE_SERVER_ON=${MESG_SERVICE_SERVER_ON:-true}
+
+MESG_SERVICE_NAME=${MESG_SERVICE_NAME:-engine}
+
 MESG_SERVICE_SERVER="engine-service-server"
+MESG_SERVER_PORT=${MESG_SERVER_PORT:-50052}
```

Comment:
This `MESG_SERVER_PORT ` config is not useful because it's changing the internal port of the engine server.

Same logic as the docker volume. The source can change, but not the destination:
```
source=$MESG_PATH,destination=/root/.mesg
```